crates/router/src/types/api/authentication.rs

```diff
@@ -0,0 +1,84 @@
+use api_models::enums;
+
+use super::BoxedConnector;
+
+#[derive(Debug, Clone)]
+pub struct PreAuthentication;
+
+#[derive(Debug, Clone)]
+pub struct Authentication;
+
+#[derive(Debug, Clone)]
+pub struct PostAuthentication;
+use crate::{services, types};
+
+#[derive(Clone, serde::Deserialize, Debug, serde::Serialize)]
+pub struct AcquirerDetails {
+    pub acquirer_bin: String,
+    pub acquirer_merchant_mid: String,
+}
+
+#[derive(Clone, serde::Deserialize, Debug, serde::Serialize)]
```

Comment:
core types need not have serialize and Deserialize

Reply:
I'll take this up in next pr